CSS/styles.css

```diff
@@ -0,0 +1,215 @@
+/* --------------------------- Base styles --------------------------- */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Arial', sans-serif;
+    -webkit-font-smoothing: antialiased; /* do I need this? */
+    background: #FFF;
+    color:#222;
+}
+
+ul {
+    list-style: none;
+}
+
+/* 
+h1, h2, h3, h3, h4 {
+
+}
+*/
+
+a {
+    color: #222;
+    text-decoration: none;
+}
+
+
+p {
+    margin: 0.5rem 0;
+}
+
+img {
+    width: 100%;
+}
+
+.content-wrap {
+    width: 90%;
+    margin: 0 auto;
+}
+
+/* Buttons */
+.btn {
+    display: inline-block;
+    cursor: pointer;
+    border: 1px solid #DDD;
+    border-radius: 4px;
+    background-color: #FFF;
+}
+
+.btn:hover {
+
+}
+
+.btn-rounded {
+	border-radius: 4px;
+}
+
+/* --------------------------- Header --------------------------- */
+header {
+    width: 100%;
+    height: 178px;
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 100;
+    background-color: #FFF;
+}
+
+.header-top {
+    height: 80px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.header-logo img {
+    width: 107px;
+}
+
+.header-buttons {
+    display: flex;
+    height: 48px;
+    text-align: center;
+    border-radius: 30px;
+    font-size: 15px;
+}
+
+.left-header-button {
+    padding: 15px 18px 18px;
+    font-weight: 500;
+}
+
+.middle-header-button {
+    padding: 15px 18px 18px;
+}
+
+.right-header-button {
+    padding: 15px 18px 18px;
+}
+
+#category-filter-bar {
+    z-index: 101;
+    display: flex;
+    justify-content: space-between;
+}
+
+#filters {
+    width: 90px;
+    height: 48px;
+    border-radius: 13px;
+}
+
+#next {
```

Comment:
There is a general web convention to avoid using ID selectors for CSS styles and really only reserve them for Javascript targets as the ID selector is VERY strong and can mess with your CSS specificity approach.

Reply:
Got it.